src/app-common/map-details-container/map-details-content.scss

```diff
@@ -0,0 +1,37 @@
+@import "../../style/sass/abstracts/variables";
+@import "../../style/sass/abstracts/mixins";
+
+.location-detail-content-container {
+  display: flex;
+  padding: 2rem;
+
+  .map-details-nav {
+    width: 12%;
+    position: relative;
+    .fixed-sticky-nav {
+      position: -webkit-sticky;
+      position: sticky;
+      top: 30%;
```

Comment:
I'm on a 15 inch screen and `top: 30%` leaves a lot of whitespace on top of the map-details-nav when scrolling. Changing it to 20% looks a lot better on my screen, but I'm guessing it makes the map-details-nav cut off when scrolling on smaller screens. In my opinion the LocationDetailHeader shouldn't be stickied, that way we can set `top: 2rem` for the map-details-nav and it'll work for all screens. Or if we do want LocationDetailHeader to be stickied, then we should calculate the `top` for map-details-nav based on the height of the LocationDetailHeader, but that makes things more complicated.

Reply:
I just pushed an update with a couple of minor tweaks. I think we do want the header to be sticky so it stays in view, but yes, I think it will mean needing to base the nav top on something else like the height of the header. I also had to add a manual scroll offset of 175px for the nav scrolling since it wasn't accounting for the height of the sticky header.

I tried to use useRef and forwardRef on the detail header to try and calculate the height, but kept getting errors. So if anyone wants to take a stab at that be my guest :-).

Reply:
Added ref for header height and scroll. Lemme know what you think.